src/models/Settings.js

```diff
@@ -87,6 +87,7 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, SharedUtil
       'features.idpDiscovery': ['boolean', false, false],
       'features.passwordlessAuth': ['boolean', false, false],
       'features.showPasswordToggleOnSignInPage': ['boolean', false, false],
+      'features.trackTypingPattern': ['boolean', false, false],
```

Comment:
where is the map from `trackTypingPattern` to the actual feature name `TYPING_PATTERN_COLLECTION`?

Reply:
That would happen in Okta-core when we bootstrap the sign in widget. That is how every feature is mapped.

Reply:
cool, thanks